recipes/ampgen/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -eux
+
+mkdir build
+cd build
+
+echo ${PREFIX}
+
+cmake \
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+    -DCMAKE_INSTALL_RPATH="${PREFIX}/lib" \
+    -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON \
+    -DAMPGENROOT_CMAKE="${PREFIX}" \
```

Comment:
This like is the problem, looks like it needs patching to something like `CMAKE_INSTALL_BINDIR`: https://github.com/GooFit/AmpGen/blob/d2c5bb112318ff0465debae9bb8dbff286a49a21/CMakeLists.txt#L167
```suggestion
```